metecho/api/models.py

```diff
@@ -1066,6 +1079,33 @@ def finalize_refresh_org(self, *, error=None, originating_user_id):
             )
             self.queue_delete(originating_user_id=originating_user_id)
 
+    def queue_reassign(self, *, new_user, originating_user_id):
+        from .jobs import user_reassign_job
+
+        self.currently_refreshing_org = True
+        self.deleted_at = None
+        self.save()
+        self.notify_list_changed(
+            type_="SCRATCH_ORG_RECREATE", originating_user_id=originating_user_id,
+        )
```

Comment:
@wlonk I'm wondering if we want to only fire this event if the org was previously `deleted_at != None`? Otherwise the org is just being re-assigned, not re-created.